### ISSUE TITLE
[Feature Request] AVIF Image Format Support

### ISSUE THREAD

**Issue by luxlogica** [bdkjones/CodeKit#628]
Now that we can see support for webp on the horizon (Issue #619), would it be possible to ask for support also for **[AVIF](https://jakearchibald.com/2020/avif-has-landed/)**? It seems to be gaining very quick adoption, because of the gigantic gains in **lossless** compression, coupled with the fact that it is already being supported and pushed forward by major players.

The current status, as of today, is that:
* Chrome supports it
* Opera supports it
* Firefox has been working on actively supporting it since January: currently avif support can be 'turned on' optionally, and it is expected to be turned on by default on new versions before the end of the year
* Edge is expected to implement it very soon, as Windows 10 already has built-in support for it, as does the Chromium engine
* Safari is expected to support it soon, as Apple was involved in the development of the AV1 codec that AVIF uses, as well as being a founding member of the 'Alliance for Open Media' that is pushing the format.

Large media and internet companies like Netflix and Cloudflare are already using it, and supporting it.

Ideally, by "support" I mean it would be great if CodeKit could actually **convert images** from other formats into AVIF...

**Comment by tavlean** [comment]
I'm transitioning all my projects from WebP to AVIF. If the conversion of images is introduced in CodeKit, I'll be very happy to become a customer.

**Comment by bdkjones** [comment]
Years ago, AVIF was poorly supported outside of Chrome. The additional file-size reduction compared to webp was statistically insignificant as well. Have those conditions changed?

**Comment by tavlean** [comment]
Absolutely. It's a successor of WebP and its adoption is increasing really fast.
Check [this](https://netflixtechblog.com/avif-for-next-generation-image-coding-b1d75675fe4) and [this](https://www.ctrl.blog/entry/webp-avif-comparison.html)

Firefox is supporting since 93, Edge is WIP. But with picture tag and using webp and jpg as a fallback, we can still serve the most optimized version to 70% of the users and webp to the other modern browsers and traditional formats for traditional browsers.
